Classes/Service/ContentService.php

```diff
@@ -283,8 +283,12 @@ protected function initializeRecordByNewAndOldAndLanguageUids($row, $newUid, $ol
 				// look for the translated version of the parent record indicated
 				// in this new, translated record. Below, we adjust the parent UID
 				// so it has the UID of the translated parent if one exists.
-				$translatedParents = (array) $this->workspacesAwareRecordService->get('tt_content', 'uid', "t3_origuid = '" . $oldRecord['tx_flux_parent'] . "'");
-				$translatedParent = reset($translatedParents);
+				if ($oldRecord['tx_flux_parent'] > 0) {
+					$translatedParents = (array)$this->workspacesAwareRecordService->get('tt_content', 'uid', "t3_origuid = '" . $oldRecord['tx_flux_parent'] . "'");
```

Comment:
Space between casting type and variable being cast. Other than that, looks good!